src/data/finset.lean

```diff
@@ -2321,6 +2404,20 @@ begin
     apply lt_of_lt_of_le H4 H6 }
 end
 
+/--
+If there's more than 1 element, the min' is different to the max'. An alternate version of
+`min'_lt_max'` which is sometimes more convenient.
+-/
+lemma min_ne_max_of_card (h₂ : 1 < card s) : s.min' H ≠ s.max' H :=
```

Comment:
This should be `min'_ne_max'_of_card`.

Reply:
And in fact I'd rather have a statement saying `min' < max'` (as it is very easy to convert a strict inequality like this to a non-equality with `ne_of_lt`. Can't you deduce your statement from `min'_lt_max'`?

Reply:
It does follow from `min'_lt_max'` but not in any quick way that I could find. I'll make the other adjustments

Reply:
I've changed the name and statement to reflect your suggestions. I think it's easier to deduce `min'_lt_max'` from my statement instead of the other way around - I can add this if you think necessary.